src/dotnet/commands/dotnet-vstest/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright(c) .NET Foundation and contributors.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+
+namespace Microsoft.DotNet.Tools.VSTest
+{
+    public class VSTestCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication cmd = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "vstest",
+                FullName = "vstest.console",
+                Description = "Test Driver for the .NET Platform"
+            };
+
+            cmd.HelpOption("-h|--help");
+
+            var arguments = cmd.Argument(
```

Comment:
It seems these "Arguments" would be better to be split out appropriately.  "The container to test and the argument with which test will run" all in one argument isn't very user-friendly. 

We should split these out into separate options.  See the other commands for examples:

`--framework netcoreapp1.0`
`--runtime win10-x64`
`--output C:\temp`
etc